src/kouta_indeksoija_service/indexer/cache/eperuste.clj

```diff
@@ -58,20 +62,33 @@
              :tunniste (:tunniste osaamisala)}
             (get-opintojen-laajuus (get-in muodostumissaanto [:laajuus :minimi])))))))
 
+(defn- get-diplomi-sisallot-tavoitteet [eperuste]
```

Comment:
parametri on näissä repoissa annettu omalla rivillää - kunhan on joku konventio jota seurataan

Reply:
Mielelläni ottasin käyttöön jonkin työkalun, joka osais formatoida tätä clojure-koodin yhdenmukaisesti (vrt. prettier), koska minua ei totisesti kiinnostaisi rivitellä näitä käsin. En löytänyt lyhyellä etsimisellä mitään vastaavaa :-(